euth_wagtail/templates/a4polls/poll_dashboard.html

```diff
@@ -0,0 +1,9 @@
+{% extends "a4dashboard/base_dashboard_project.html" %}
+{% load i18n react_polls %}
+
+{% block dashboard_project_content %}
+    <h1 class="u-first-heading">{% trans 'Edit poll' %}</h1>
+
+    {% react_poll_form poll True %}
```

Comment:
I actually have no idea. The True should reload it, and it's also included like that in a4, soooo, hmmm.

Reply:
well good to know that's what you meant! :) it's included twice in mB, but that is a different structure, i feel like this is a morning brain thing so will do bootstrap issues for rest of day and come back to this in morning :)

Reply:
This commit can be dropped - the a4 template works just fine